client/landing/gutenboarding/components/plans/plans-table/plan-item.tsx

```diff
@@ -66,7 +66,7 @@ const PlanItem: React.FunctionComponent< Props > = ( {
 					{ isPopular && <div className="plan-item__badge">{ __( 'Popular' ) }</div> }
 				</div>
 				<div className="plan-item__price">
-					<div className="plan-item__price-amount" data-is-loading={ ! price }>
+					<div className={ classNames( 'plan-item__price-amount', { 'is-loading': ! price } ) }>
```

Comment:
No logic changes here, just use classes instead of data attribs because IE is buggy around data-attributes and CSS selectors.